qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +60,42 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
```

Comment:
Can you add a new format to the dummy plugin that doesn't have a transformer mapping it to a canonical format? That format shouldn't show up in the set of importable formats.